base/applications/taskmgr/lang/it-IT.rc

```diff
@@ -413,19 +413,19 @@ BEGIN
     IDS_MENU_DETAILS "&Dettagli"
     IDS_MENU_ONEGRAPHALLCPUS "&Un grafico, tutte le CPU"
     IDS_MENU_ONEGRAPHPERCPU "Un grafico &Per CPU"
-    IDS_MENU_CPUHISTORY "&CPU History"
+    IDS_MENU_CPUHISTORY "Storia &CPU"
     IDS_MENU_SHOWKERNELTIMES "&Mostra tempo di Kernel"
-    IDS_CREATENEWTASK "Create Nuovo Task"
-    IDS_CREATENEWTASK_DESC "Scrivi il nome di un programma, Cartella, documento o risorsa internet e Task Manager lo aprirà."
+    IDS_CREATENEWTASK "Crea nuovo processo"
+    IDS_CREATENEWTASK_DESC "Scrivi il nome di un programma, Cartella, documento o risorsa internet e Gestione attività lo aprirà."
```

Comment:
You changed that string back to "Task Manager" in several places, but you forgot to do the same here?

Reply:
thanks for flagging that, indeed I forgot to change some strings back.